### PR TITLE
task was done

### PR DESCRIPTION
**Pull request by vanyachyzh**
- [DEMO LINK](https://vanyachyzh.github.io/js_promises_practice_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1 +1,72 @@
 'use strict';
+
+const BODY = document.body;
```
Comment (OlEzhA0) [inline src/scripts/main.js:3]:
```suggestion
const body = document.body;
```

File: src/scripts/main.js [hunk 2]
```diff
@@ -1 +1,72 @@
...
+
+function showSuccess(result) {
+  const MESSAGE = document.createElement('div');
```
Comment (OlEzhA0) [inline src/scripts/main.js:6]:
```suggestion
  const message = document.createElement('div');
```

Comment (OlEzhA0) [reply src/scripts/main.js:6]: check it for each your constant

File: src/scripts/main.js [hunk 3]
```diff
@@ -1 +1,72 @@
...
+    resolve('Second promise was resolved');
+  });
+
+  BODY.addEventListener('contextmenu', () => {
+    resolve('Second promise was resolved');
+  });
+});
+
+const PROMISE3 = new Promise((resolve, reject) => {
+  let checker = '';
```
Comment (OlEzhA0) [inline src/scripts/main.js:46]: you can create 2 boolean variables like
`let leftClick = false`
`let rightClick = false`
and make it true when clicked

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
hey! if you have any questions - ping me in fe chat

**Review by slavanosatskyi — APPROVED** [review]

**Pull request closed without merging**